src/detect-engine-port.c

```diff
@@ -1465,13 +1471,10 @@ DetectPort *PortParse(const char *str)
  */
 int DetectPortIsValidRange(char *port)
```

Comment:
you could declare it static (so that we see that these changes do not affect other files)

Reply:
Yes OK.